src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
Action names are suitable for methods, interfaces naming is better as noun or adjective, e.g. Drawable